### DISCUSSION TITLE
HM-MOD-RPI-PCB an Raspberrymatric

### DISCUSSION THREAD

**Discussion by opl-git** [OpenCCU/OpenCCU#2958] [🇩🇪 Fragen&Antworten]
Hallo zusammen,

ich nutze aktuell ein HM-MOD-RPI-PCB an einem ESP8266 als WiFi Gateway in FHEM. Läuft mit esp-link v2.2.3. 
Nun möchte ich auf Home Assistant umsteigen,  welches in einer VM unter Proxmox läuft.
Kann ich den WiFi Gateway weiter nutzen?
Hatte auch schon probiert den HM-MOD-RPI-PCB mit einem FTDI über USB in Raspberrymatric zu nutzen. Habe dieses Setup aber leider nicht zum Laufen gebracht.
Vielleicht hat jemand Tipps für mich.
Besten Dank!

**Comment by teichtaucher** [comment]
Hi, ich habe genau das gleiche Setup und war auf der Suche nach der gleichen Lösung für unser Problem. Meine Lösung mit dem höchsten Re-Use: Nimm einen alten RaspberryPi, installiere RaspberryMatic, konfiguriere es als LAN Gateway und verbinde das dann mit der RaspberryMatic Instanz auf Proxmox. Ich hatte noch einen Pi Zero W rumfliegen - Läuft jetzt.
Allerdings versuche ich gerade die HM-ID zu ändern, so dass ich die Geräte einfacher migrieren kann. Ich würde gern die HM ID aus FHEM übernehmen. So wie hier beschrieben: https://homematic-forum.de/forum/viewtopic.php?f=65&t=75987
